code/__DEFINES/preferences.dm

```diff
@@ -27,8 +27,9 @@
 #define DONATOR_PUBLIC	32768
 #define CHAT_NO_TICKETLOGS 65536
 #define UI_DARKMODE 131072
+#define KARMA_TOGGLE 262144
 
-#define TOGGLES_DEFAULT (CHAT_OOC|CHAT_DEAD|CHAT_GHOSTEARS|CHAT_GHOSTSIGHT|CHAT_PRAYER|CHAT_RADIO|CHAT_LOOC|MEMBER_PUBLIC|DONATOR_PUBLIC)
+#define TOGGLES_DEFAULT (CHAT_OOC|CHAT_DEAD|CHAT_GHOSTEARS|CHAT_GHOSTSIGHT|CHAT_PRAYER|CHAT_RADIO|CHAT_LOOC|MEMBER_PUBLIC|DONATOR_PUBLIC|KARMA_TOGGLE)
```

Comment:
I don't think this default is applied to all existing players.

So, on merge of this PR, either a SQL query will have to be done to update all existing players... or all existing players will find their ability to get karma disabled.

You can avoid this by making the KARMA_TOGGLE into a DISABLE_KARMA flag that is not default... that way existing players don't have it applied but people can toggle it on.